Resources/Server Info/Guidebook/Chef/pitzah.xml

```diff
@@ -96,4 +103,102 @@
 <GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
 <GuideEntityEmbed Entity="FoodPizzaDonkpocket" Caption=""/>
 </Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Margherita Pizza[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- Mozzarella
+- Cheese wedge
+- Ambrosia Vulgaris
+</Box>
+<Box Orientation="Vertical">
+- Dough
+- 2 Tomatoes
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
+<GuideEntityEmbed Entity="FoodMothPizzaMargherita" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Firecracker Pizza[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- 10u BBQ sauce
+- Cheese wedge
+- Baked Corn
+</Box>
+<Box Orientation="Vertical">
+- Dough
+- Chili pepper
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="30 sec"/>
+<GuideEntityEmbed Entity="FoodMothPizzaFirecracker" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Mothic Five-Cheese Pizza[/color]
```

Comment:
It's only got four cheeses in it. 3.5, really since curd cheese and cheese curd is still two pieces of curdy cheese where one was heated in a microwave.